index.js

```diff
@@ -15,7 +15,7 @@ try {
   const comment = payload.comment ? payload.comment.body : payload.issue.body;
   console.log(`comment: ${comment}`)
   const exemptDomainsInput = core.getInput('exemptions')
-  const exemptDomains = exemptDomainsInput.split(',').map(d => d.toLowerCase()).map(d => '@' + d);
+  const exemptDomains = exemptDomainsInput.split(',').map(d => d.toLowerCase()).map(d => d);
```

Comment:
Remove `.map(d => d)`

Reply:
thank you for reviewing this sir @IvanPizhenko . I just removed the unnecessary code.